src/reducers/index.js

```diff
@@ -30,6 +30,9 @@ let reducer = (store = initialStore, action) => {
       localStorage.removeItem("session");
       return Object.assign({}, store, { isLoggedIn: false });
 
+    case TOGGLE:
+      return Object.assign({}, store, { display: action.payload });
+
```

Comment:
I don't see this chain of functions being used anywhere and the component works fine, so I think it can be removed.

Reply:
removed